tools/toolchain/README.md

```diff
@@ -70,7 +70,7 @@ If you add dependencies (to `install-dependencies.sh` or
 
 Run the command
 
-    podman build --no-cache --pull -f tools/toolchain/Dockerfile .
+    podman build --no-cache --pull -f tools/toolchain/Dockerfile . -v "$(realpath ./):/mnt:Z" -t $(<tools/toolchain/image)
 
```

Comment:
I don't understand why this is needed.

Reply:
The Dockerfile we have for the toolchain (https://github.com/scylladb/scylladb/blob/master/tools/toolchain/Dockerfile), does not have any ADD/COPY directives but expects the scripts to be there, meaning the build will simply fail without it (At least with Podman). In the `prepare` script we use this exact mount and tag (https://github.com/scylladb/scylladb/blob/88ab8db94428e8289159dd6e7a526630c77075ca/tools/toolchain/prepare#L186), so I just copied it to the documentation as well

Reply:
I don't understand. I never used this /mnt mount.

Reply:
I started to investigate this after the original command `podman build --no-cache --pull -f tools/toolchain/Dockerfile .` failed for me, due to missing install scripts, does it work for you? If so, with what version of podman?

Reply:
I use the prepare script, which in turn uses buildah.

Reply:
Right, and that prepare script does use /mnt mount (`-v "$(realpath ./):/mnt:Z"`), which is why prepare work, but the current documented command using pure Podman does not, which is why I am bringing the documentation on par with what is being done in prepare script

Reply:
I see. But I don't understand that bind mount.

Maybe it's for the clang build? @syuu1228 ?